consensus/istanbul/backend/announce.go

```diff
@@ -256,10 +255,10 @@ func (sb *Backend) handleIstAnnounce(payload []byte) error {
 		return err
 	}
 
-	logger = logger.New("msgAddress", msg.Address, "msg_round", announceData.View.Round, "msg_seq", announceData.View.Sequence)
+	logger = logger.New("msgAddress", msg.Address, "msg_timestamp", announceData.Timestamp)
 
-	if view, err := sb.valEnodeTable.GetViewFromAddress(msg.Address); err == nil && announceData.View.Cmp(view) <= 0 {
-		logger.Trace("Received an old announce message", "senderAddr", msg.Address, "messageView", announceData.View, "currentEntryView", view)
+	if currentEntryTimestamp, err := sb.valEnodeTable.GetTimestampFromAddress(msg.Address); err == nil && announceData.Timestamp < currentEntryTimestamp {
+		logger.Trace("Received an old announce message", "senderAddr", msg.Address, "messageTimestamp", announceData.Timestamp, "currentEntryTimestamp", currentEntryTimestamp)
```

Comment:
nit: `senderAddr` and `messageTimestamp` are redundant since we add them in the logger on line 258